app/views/subscriptions/new_address.html.erb

```diff
@@ -29,7 +29,7 @@
       <%= t("subscriptions.new_address.summary.#{@frequency}") %>
     </p>
 
-    <%= form_tag verify_subscription_path, method: :post, class: "checklist-email-signup", novalidate: "novalidate" do %>
```

Comment:
I think we need the `novalidate: "novalidate"` so we dont show the default browser error if someone tries to submit the form without an email address?